src/cluster/kv/etcd/store.go

```diff
@@ -103,17 +103,15 @@ func NewStore(etcdKV clientv3.KV, etcdWatcher clientv3.Watcher, opts Options) (k
 
 	if store.cacheFile != "" {
 		if err := store.initCache(); err != nil {
-			store.logger.Warnf("could not load cache from file %s: %v", store.cacheFile, err)
+			store.logger.Warn("could not load cache from file", zap.String("file", store.cacheFile), zap.Error(err))
 		} else {
-			store.logger.Infof("successfully loaded cache from file %s", store.cacheFile)
+			store.logger.Info("successfully loaded cache from file", zap.String("file", store.cacheFile))
 		}
 
 		go func() {
 			for range store.cacheUpdatedCh {
 				if err := store.writeCacheToFile(); err != nil {
-					store.logger.
-						WithFields(log.NewErrField(err)).
-						Error("failed to write cache file")
+					store.logger.With(zap.Error(err)).Error("failed to write cache file")
```

Comment:
nit: `failed to write cache file` => `failed to write cache to file`?

Reply:
sure.